RELEASING.md

```diff
@@ -53,20 +53,14 @@ your `~/.sbt/1.0/private-credentials.sbt`.
 [Graphvis](https://graphviz.gitlab.io/download/) is needed for the 
 scaladoc generation build task, which is part of the release.
  
-### Release script instructions
-
-Make sure you have completed the setup in `project/scripts/release`.
-
 ## Snapshot releases
 
-Nightly snapshot releases are created from master and published to
-https://repo.akka.io/snapshots by https://jenkins.akka.io:8498/job/akka-publish-nightly/
+Snapshot releases are created from master and published to
+https://oss.sonatype.org/content/repositories/snapshots/com/typesafe/akka/
```

Comment:
could you also update the doc page about snapshots https://doc.akka.io/docs/akka/current/project/links.html#snapshots-repository